torch/utils/collect_env.py

```diff
@@ -458,6 +592,7 @@ def get_env_info():
         debug_mode_str = str(torch.version.debug)
         cuda_available_str = str(torch.cuda.is_available())
         cuda_version_str = torch.version.cuda
+        xpu_available_str = str(torch.xpu.is_available())
```

Comment:
Now, we can use `torch.version.xpu` here.

Reply:
added.